### PR TITLE
argo-cd-sync for GitOps Runtimes

### PR DESCRIPTION
**Pull request by francisco-codefresh**

### CODE REVIEW

File: incubating/argo-cd-sync/step.yaml
Comment (lrochette) [inline incubating/argo-cd-sync/step.yaml:1]: Do we still use that URL for icon?
I have been using something like https://raw.githubusercontent.com/codefresh-io/steps/master/incubating/aws-s3/image/s3-logo.svg lately
@eti-codefresh ?

File: incubating/argo-cd-sync/argocd_sync.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+from gql import Client, gql
+from gql.transport.requests import RequestsHTTPTransport
+import os
+
```
Comment (lrochette) [inline incubating/argo-cd-sync/argocd_sync.py:4]: Should we add some kind of logging code?
and a LOG_LEVEL option?

**Review by lrochette — APPROVED** [review]

File: incubating/argo-cd-sync/step.yaml
Comment (francisco-codefresh) [reply incubating/argo-cd-sync/step.yaml:1]: I took it from our legacy argo cd sync step:
https://github.com/codefresh-io/steps/blob/lrochette-argo-sync-gitops/graduated/argocd-sync/step.yaml#L16

Comment (francisco-codefresh) [reply incubating/argo-cd-sync/step.yaml:1]: i can change it in the next version :)

File: incubating/argo-cd-sync/argocd_sync.py (hunk 1 above)
Comment (francisco-codefresh) [reply incubating/argo-cd-sync/argocd_sync.py:4]: Definitely. It is something to add to the next version

**Pull request merged**